Source/API/EbApi.h

```diff
@@ -100,6 +101,8 @@ extern "C" {
         uint8_t* naluBase64Encode;
 
         SegmentOverride_t *segmentOvPtr;
+
+        LIST_ENTRY(EB_BUFFERHEADERTYPE) list;
```

Comment:
Since there is no utility you can use on Windows, and the linked list is only used in app layer. Let's write our own linked list in app layer to manage the buffer.